### DISCUSSION TITLE
Potential issue with boost timers in Duke room

### DISCUSSION THREAD

**Discussion by tjucf** [runelite/runelite#16866] [Q&A]
I'm not sure if this is a runelite bug or something with the Duke fight, but I notice the boost timer doesn't correctly count down, it just resets itself at 29 seconds. So it goes 59->29, resets to 59, then at 30 the actual boost tick down happens, resetting the countdown to 59. Attached example gif https://i.imgur.com/5jsvxhz.gif

**Comment by Alexsuperfly** [comment]
do you happen to have the preserve prayer active

**Reply by tjucf** [reply]
No
